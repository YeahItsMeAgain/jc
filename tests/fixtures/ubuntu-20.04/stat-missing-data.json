[{"file": "/etc/passwd", "size": 3399, "blocks": 8, "io_blocks": 4096, "type": "regular file", "device": "810h/2064d", "inode": 6392, "links": 1, "access": "0644", "flags": "-rw-r--r--", "uid": 0, "user": "root", "gid": 0, "group": "root", "access_time": "2023-06-01 13:12:52.776423000 -0700", "modify_time": "2023-01-07 03:10:13.752410800 -0800", "change_time": "2023-01-07 03:10:13.752410800 -0800", "access_time_epoch": 1685650372, "access_time_epoch_utc": null, "modify_time_epoch": 1673089813, "modify_time_epoch_utc": null, "change_time_epoch": 1673089813, "change_time_epoch_utc": null}]