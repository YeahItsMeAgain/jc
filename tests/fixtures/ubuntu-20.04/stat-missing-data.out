  File: /etc/passwd
  Size: 3399            Blocks: 8          IO Block: 4096   regular file
Device: 810h/2064d      Inode: 6392        Links: 1
Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2023-06-01 13:12:52.776423000 -0700
Modify: 2023-01-07 03:10:13.752410800 -0800
Change: 2023-01-07 03:10:13.752410800 -0800